### PR TITLE
[RHACS] Update docs with correct variable info for wildcard Prometheus monitoring

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):

Merge to:
- `rhacs-docs`

Cherry pick to:
- `rhacs-docs-3.72`
- `rhacs-docs-3.71`
- `rhacs-docs-3.70`

[Issue](https://issues.redhat.com/browse/ROX-12478)

[Link to docs preview](https://openshift-docs-fx9jsxesm-kcarmichael08.vercel.app/openshift-acs/master/configuration/monitor-acs.html)

### CODE REVIEW

File: modules/enable-monitoring-customize-port.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ oc -n stackrox set env deploy/central ROX_METRICS_PORT=<value> <1>
 You can specify the `<value>` for the `ROX_METRICS_PORT` environment variable as:
 
 * `disabled` to disable monitoring.
-* `<port_number>` to bind it to a wildcard address.
+* `0.0.0.0:<port_number>` to bind it to a wildcard address.
```
[the remark this answers is not in the document]
Comment (misberner) [reply modules/enable-monitoring-customize-port.adoc:25]:
```suggestion
* `:<port_number>` to bind it to a wildcard address.
```
is what should be stated here, nothing else

**Review by misberner — CHANGES_REQUESTED** [review]

**Comment by pneedle-rh** [comment]
LGTM

**Pull request merged**